src/main/webapp/app/assessment/structured-grading-instructions-assessment-layout/structured-grading-instructions-assessment-layout.component.ts

```diff
@@ -64,4 +93,12 @@ export class StructuredGradingInstructionsAssessmentLayoutComponent implements O
     disableDrag() {
         return this.allowDrop;
     }
+
+    collectCollapsableSections() {
+        setTimeout(() => {
```

Comment:
Set timeout as described in the official angular example: https://angular.io/api/core/ViewChildren#another-example

Reply:
Could you add a comment in the code as well why we need this timeout here?